### PR TITLE
feat: support dynamic import in script loader

### LINKED ISSUE #18: Use native `import()` calls
I just tried `import(chrome.runtime.getURL())` and it seems to work both in Chrome and Firefox, in content scripts and background pages 🎉  

This is because the bug that your readme points to, has been marked as solved and shipped in the latest Firefox release: https://bugzilla.mozilla.org/show_bug.cgi?id=1536094

Could this plugin be modified to use `import()` instead of the current messaging mechanism? 

Does Webpack even leave native `import()` calls in the bundle? In my test, `import()` is still converted to Webpack’s own loading mechanism and I don't see anything about raw `import()` calls in the docs: 

https://webpack.js.org/guides/code-splitting/#dynamic-imports
https://webpack.js.org/api/module-methods/#import-1

### PR DESCRIPTION
**Pull request by Jack-Works**
close #18

and @crimx can you add me to the maintainer of this repo so I can push to this repo directly?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ Native dynamic import is [buggy](https://bugzilla.mozilla.org/show_bug.cgi?id=15
 
 The Firefox addons-linter is also [making aggressive errors](https://github.com/mozilla/addons-linter/issues/2498) on dynamic import. A workaround is to just replace the `import` with other name. Since all the dynamic chunks are loaded in Firefox the `import()` code should never be run.
 
+In webpack 5, you need to enable `output.environment.dynamicImport` to *true*.
```
Comment (Jack-Works) [inline README.md:23]: cc @fregante

**Review by crimx — APPROVED** [review]

**Comment by crimx** [comment]
Thanks! I am thinking about transferring this repo to a organization. How about we name it `awesome-webextension`?

**Comment by Jack-Works** [comment]
> Thanks! I am thinking about transferring this repo to a organization. How about we name it `awesome-webextension`?

Cool

**Comment by crimx** [comment]
Do you receive the invitation?

**Comment by Jack-Works** [comment]
> Do you receive the invitation?

yes thanks!

**Comment by crimx** [comment]
Added.

**Pull request merged**